### PR TITLE
samples: wifi: sta: Fix ground connections

### PR DESCRIPTION
**Pull request by krish2718**
We cannot use the same ground for both PPKs, so, use P4-Pin7 for ground (using berg connector).

Also, add a figure to demonstrate the setup.

Signed-off-by: Krishna T <krishna.t@nordicsemi.no>

@richabp  It would be good if we can update the figure that I have copied from power-save docs to reflect two PPK connections, for now I have used the same as a place holder. @busineni can help you with any doubts.

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

File: samples/wifi/sta/README.rst [hunk 1]
```diff
@@ -187,9 +187,16 @@ To measure the current consumption of the nRF7002 device, complete the following
 
  * Remove jumper on **P23** (VBAT jumper).
  * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as  **GND** (-).
+    You can use **P4** pin **7** mentioned as  **GND** (-).
```
Comment (busineni) [inline samples/wifi/sta/README.rst:190]: Remove (-) in this the line. 
In line number 183 please make as **-** for GND

**Review by busineni — CHANGES_REQUESTED** [review]

**Review by busineni — APPROVED** [review]

File: samples/wifi/sta/README.rst [hunk 2]
```diff
@@ -180,16 +180,18 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
  * Cut solder bridge **SB16**.
  * Make a short on solder bridge **SB17**.
  * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as **GND** (-).
+    You can use **P4** pin **7** mentioned as **GND** for ground.
+    Please note this needs a berg pin connector.
```
Comment (richabp) [inline samples/wifi/sta/README.rst:184]:
```suggestion
    Note that this connection requires a berg pin connector.
```

**Review by richabp — CHANGES_REQUESTED** [review]

File: samples/wifi/sta/README.rst [hunk 3]
```diff
@@ -180,16 +180,18 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
  * Cut solder bridge **SB16**.
  * Make a short on solder bridge **SB17**.
  * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as **GND** (-).
+    You can use **P4** pin **7** mentioned as **GND** for ground.
```
Comment (richabp) [inline samples/wifi/sta/README.rst:183]:
```suggestion
   You can use **P4** pin **7** mentioned as **GND** for ground.
```

File: samples/wifi/sta/README.rst [hunk 4]
```diff
@@ -180,16 +180,18 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
...
+    Please note this needs a berg pin connector.
  * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.
 
 To measure the current consumption of the nRF7002 device, complete the following steps:
 
  * Remove jumper on **P23** (VBAT jumper).
  * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as  **GND** (-).
+    You can use **P21** pin **1** mentioned as **-** for ground.
```
Comment (richabp) [inline samples/wifi/sta/README.rst:191]:
```suggestion
   You can use **P21** pin **1** mentioned as **-** for ground.
```

**Review by richabp — CHANGES_REQUESTED** [review]

File: samples/wifi/sta/README.rst (hunk 3 above)
Comment (richabp) [reply samples/wifi/sta/README.rst:183]: The formatting needs to be corrected as it makes lines 182 and 190 **bold**.

**Review by bama-nordic — APPROVED** [review]

File: samples/wifi/sta/README.rst (hunk 3 above)
Comment (richabp) [reply samples/wifi/sta/README.rst:183]: No. The lines which are appearing in bold needs to be in regular font.

File: samples/wifi/sta/README.rst [hunk 5]
```diff
@@ -179,21 +179,40 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
  * Remove jumper on **P22** (VDD jumper).
  * Cut solder bridge **SB16**.
  * Make a short on solder bridge **SB17**.
- * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as **GND** (-).
- * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.
+ * Connect **GND** on PPK2 kit to **GND** on the nRF5340 SoC.
```
Comment (busineni) [inline samples/wifi/sta/README.rst:182]: Replace nRF5340 with nRF7002 DK

File: samples/wifi/sta/README.rst [hunk 6]
```diff
@@ -179,21 +179,40 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
...
+
+     You can use **P4** pin **7** mentioned as **GND** for ground.
+
+     Note that this connection requires a berg pin connector.
+
+ * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF5340 SoC.
```
Comment (busineni) [inline samples/wifi/sta/README.rst:188]: Replace nRF5340 with nRF7002 DK

File: samples/wifi/sta/README.rst [hunk 7]
```diff
@@ -179,21 +179,40 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
...
 
 To measure the current consumption of the nRF7002 device, complete the following steps:
 
  * Remove jumper on **P23** (VBAT jumper).
- * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as  **GND** (-).
- * Connect the **Vout** on PPK2 to **P23** pin **1** on the nRF7002 DK.
+ * Connect **GND** on PPK2 kit to **GND** on the nRF7002 device.
```
Comment (busineni) [inline samples/wifi/sta/README.rst:193]: Replace nRF7002 device with nRF7002 DK

File: samples/wifi/sta/README.rst [hunk 8]
```diff
@@ -179,21 +179,40 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
...
+
+     You can use **P21** pin **1** mentioned as **-** for ground.
+
+ * Connect the **Vout** on PPK2 to **P23** pin **1** on the nRF7002 device.
```
Comment (busineni) [inline samples/wifi/sta/README.rst:197]: Replace nRF7002 device with nRF7002 DK

**Review by busineni — CHANGES_REQUESTED** [review]

File: samples/wifi/sta/README.rst (hunk 6 above)
Comment (krish2718) [reply samples/wifi/sta/README.rst:188]: No, this was intentional, DK covers both 5340 SoC + 7002 device, so, to clarify we are using individual component names.

File: samples/wifi/sta/README.rst (hunk 5 above)
Comment (krish2718) [reply samples/wifi/sta/README.rst:182]: No, this was intentional, DK covers both 5340 SoC + 7002 device, so, to clarify we are using individual component names.

File: samples/wifi/sta/README.rst (hunk 6 above)
Comment (krish2718) [reply samples/wifi/sta/README.rst:188]: Sorry, after re-reading, I understand your intention, will fix.

**Review by busineni — APPROVED** [review]

File: samples/wifi/sta/README.rst [hunk 9]
```diff
@@ -174,27 +174,45 @@ The nRF5340 SoC is connected to the first PPK2 and the nRF7002 DK is connected t
 Hardware modifications
 ======================
 
-To measure the current consumption of the nRF5340 SoC, complete the following steps:
+To measure the current consumption of the nRF5340 SoC in nRF7002 DK, complete the following steps:
```
Comment (richabp) [inline samples/wifi/sta/README.rst:177]:
```suggestion
To measure the current consumption of the nRF5340 SoC in the nRF7002 DK, complete the following steps:
```

File: samples/wifi/sta/README.rst [hunk 10]
```diff
@@ -174,27 +174,45 @@ The nRF5340 SoC is connected to the first PPK2 and the nRF7002 DK is connected t
...
+
+   .. figure:: ../../../doc/nrf/images/nrf7002_nrf5340_current_measurements.svg
+      :alt: Typical configuration for measuring current on the nRF7002 DK for both the nRF5340 SoC and the nRF7002 device.
+
+      Typical configuration for measuring current on the nRF7002 DK for both the nRF5340 SoC and the nRF7002 device.
+
+PPK2 usage and measurement
+==========================
+
+To measure the current consumption of the nRF5340 SoC and nRF7002 device, complete the following steps:
```
Comment (richabp) [inline samples/wifi/sta/README.rst:208]:
```suggestion
To measure the current consumption of the nRF5340 SoC and the nRF7002 device, complete the following steps:
```

File: samples/wifi/sta/README.rst [hunk 11]
```diff
@@ -174,27 +174,45 @@ The nRF5340 SoC is connected to the first PPK2 and the nRF7002 DK is connected t
...
+
+* Configure PPK2 connected to nRF5340 SoC as a source meter with 1.8 volts.
+* Configure PPK2 connected to nRF7002 device as a source meter with 3.6 volts.
```
Comment (richabp) [inline samples/wifi/sta/README.rst:211]:
```suggestion
* Configure PPK2 connected to the nRF5340 SoC as a source meter with 1.8 volts.
* Configure PPK2 connected to the nRF7002 device as a source meter with 3.6 volts.
```

**Review by richabp — APPROVED** [review]

**Pull request merged**